registry/api/v2/errors.go

```diff
@@ -40,16 +40,6 @@ var (
 		HTTPStatusCode: http.StatusBadRequest,
 	})
 
-	// ErrorCodeSizeInvalid is returned when uploading a blob if the provided
-	ErrorCodeSizeInvalid = errcode.Register(errGroup, errcode.ErrorDescriptor{
```

Comment:
This is a specification change and requires an update to the docs/specs/api.md. Please submit in a separate PR.